Assets/Scripts/Turret/TurretStats.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class TurretStats : MonoBehaviour
+    {
+        public float speed;
+        public float angleRange;
+        public float hp;
+        public float fireRate;
+        public float power;
+
+        void Start()
+        {
+            speed = 40f;
+            angleRange = 160f;
+            hp = 100f;
+            fireRate = 2f;
+            power = 12.5f;
```

Comment:
Remove `Start()` put values on properties so anyone can change them from the inspector.